tests/main/views/test_briefs.py

```diff
@@ -1504,18 +1506,19 @@ def test_can_supply_award_details_for_closed_brief_with_awarded_brief_response(s
             db.session.add(brief_response)
             db.session.commit()
             assert brief_response.status == 'pending-awarded'
+            brief_response_id = brief_response.id
```

Comment:
I'm not sure I see the benefit of introducing a new variable?

Reply:
`brief_response.id` becomes stale once the request is made to the view. You can't access it any more.

Reply:
What makes the object become stale?
Is it the new context being used? Or is it within the new context, a change is committed to the DB?

Reply:
> Is it the new context being used
Yeah, as soon as a new context is introduced the old one is stale